lib/mayaUsd/fileio/primUpdaterManager.cpp

```diff
@@ -1,5 +1,5 @@
 //
-// Copyright 2021 Autodesk
```

Comment:
Q: is it normal practice to update this? The copyright began in 2021. Just curious.